packages/legacy-client/src/client/net/executeAction.ts

```diff
@@ -71,6 +77,44 @@ export async function executeAction<
   );
 }
 
+export async function executeBatchAction<
+  O extends OntologyDefinition<any>,
+  A extends keyof O["actions"],
+  Op extends BulkActionExecutionOptions,
+>(
+  client: ClientContext<OntologyDefinition<any>>,
+  actionApiName: A,
+  params?: ActionArgs<O, A>[],
+  options?: Op,
+): WrappedBulkActionReturnType<O, A, Op> {
+  return wrapResult(
+    async () => {
+      const response = await applyActionBatchV2(
+        createOpenApiRequest(client.stack, client.fetch),
+        client.ontology.metadata.ontologyApiName,
+        actionApiName as string,
+        {
+          requests: params
+            ? remapBulkActionParams<O, A>(params)
+            : [],
```

Comment:
Given empty array is used here, does this mean that any actions that take zero arguments can never be called in bulk? If thats the intention, should we just exclude them in the type mapping for creating bulk actions?